starter-code/javascript/chronometer.js

```diff
@@ -1,39 +1,53 @@
 // Constructor
-// function Chronometer() {
-
-// }
-
-// Chronometer.prototype.startClick = function () {
-
-// };
-
-// Chronometer.prototype.setMinutes = function () {
-  
-// };
-
-// Chronometer.prototype.setSeconds = function () {
-  
-// };
-
-// Chronometer.prototype.twoDigitsNumber = function () {
-  
-// };
-
-// Chronometer.prototype.setTime = function () {
-
-// };
+  function Chronometer() {
+    this.currentTime = 0;
+    this.intervalId =0;
+}
+
+Chronometer.prototype.startClick = function(){
+  var self = this;
+
+  function increment(){
```

Comment:
next step: declaring the callback inside the setInterval. It will be something like:
```
this.intervalId = setInterval(function() {
     self.currentTime += 1;
     self.setTime();
}, 1000);
```